packages/@okta/vuepress-site/docs/concepts/session/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Session management
+meta:
+  - name: description
+    content: An overview of session management in Okta. Learn about sessions in Okta and what kind of session sign out to implement.
+---
+
+# Session management with Okta
+
+An Identity Provider (IdP) is a service that stores and manages digital identities. Okta is an IdP service that stores and verifies user identities. Okta provides a federated sign-in service so that a user can access multiple applications with a single set of sign-in credentials. A session is a group of user interactions with a resource, such as an IdP service or an application (app), for a given amount of time. When you use Okta as your IdP, there are two types of sessions that you need to consider for managing the user experience&mdash;the IdP session and the application session.
```

Comment:
```suggestion
An Identity Provider (IdP) is a service that stores and manages digital identities. Okta is an IdP service that stores and verifies user identities. Okta provides a federated sign-in service so that a user can access multiple applications with a single set of sign-in credentials. A session is a group of user interactions with a resource, such as an IdP service or an application (app) for a given amount of time. When you use Okta as your IdP, there are two types of sessions that you need to consider for managing the user experience&mdash;the IdP session and the application session.
```

Reply:
I added the comma to close the parentheses phrase. I guess I can just use parentheses here to make the additional info less formal... but I was avoiding the double parentheses since I have "(app)" already.